constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -1857,6 +1859,28 @@ protected static Task createTaskResponse(String string) {
         return new Gson().fromJson(transformed, Task.class);
     }
 
+    /**
+     * Transforms a JSON string to a new JSON string for easy Gson parsing into a Next Quiz response.
+     * Using JSON objects to achieve this is considerably less error prone than attempting to do it in
+     * a Gson Type Adapter.
+     */
+    protected static NextQuizResponse createNextQuizResponse(String string) {
```

Comment:
Should we make some adjustments to make this more consistent with the API and the other SDKs?
* `NextQuizResponse` -> `NextQuestionResponse`
* `createNextQuizResponse` -> `createNextQuestionResponse`